tools/test-framework/src/docs/walkthroughs/simple.rs

```diff
@@ -0,0 +1,65 @@
+//! ## Example Test
+//!
+//! Here's an example of a simple test that showcases some of the important features that the
+//! framework exposes:
+//!
+//! ```no_run
+//! use ibc_integration_test::prelude::*;
+//!
+//! pub struct ExampleTest;
+//!
+//! #[test]
+//! pub fn example_test() -> Result<(), Error> {
+//!     run_binary_channel_test(&ExampleTest)
+//! }
+//!
+//! impl TestOverrides for ExampleTest {}
+//!
+//! impl BinaryChannelTest for ExampleTest {
+//!     fn run<ChainA: ChainHandle, ChainB: ChainHandle>(
+//!         &self,
+//!         _config: &TestConfig,
+//!         _relayer: RelayerDriver,
+//!         _chains: ConnectedChains<ChainA, ChainB>,
+//!         _channel: ConnectedChannel<ChainA, ChainB>,
+//!     ) -> Result<(), Error> {
+//!         suspend()
+//!     }
+//! }
+//! ```
+//!
+//! The `example_test` function is the top-most function that drives the test itself. The top-
+//! most test function typically does nothing more than to specify what kind of test we want
+//! to run. In the case of this example, we're running a
+//! [`BinaryChannelTest`](ibc_test_framework::framework::binary::channel::BinaryChannelTest),
```

Comment:
There are warnings in broken cargo doc links. I think we have to use `crate::` here, since we are referring to the same crate.